x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -162,8 +162,12 @@ bulk request on an index or data stream that contains new fields that may
 be mapped rather than an explicit <<indices-put-mapping,put mapping>> request.
 
 `create`::
-Privilege to index documents. Also grants access to the update mapping
-action.
+Privilege to index documents, allowing overwriting any existing document, but
+not permitting updating one.
+_Deprecated:_ Also grants the permission to update the index mapping (but not
```

Comment:
Just a minor formatting change.
```suggestion
not permitting updating one.
+
deprecated:[7.9] Also grants the permission to update the index mapping (but not
```

Reply:
Aargh, I knew about the `deprecated:[x.x]` marker.